src/components/Schedule/ReservationInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from "react";
+
+interface IProps {
+  date: string;
+  therapyName: string;
+  doctorName: string;
+  place: string;
+  therapyLength: number;
+  price: number;
+}
+
+export const ReservationInfo: FC<IProps> = ({
+  date,
+  therapyName,
+  doctorName,
+  place,
+  therapyLength,
+  price
+}) => (
+  <div className="pt-16">
+    <div className="flex">
```

Comment:
you can apply the font gilbert class here, and then you don't have to write it twice in the children of this div